lib/api/apiUtils/object/objectRestore.js

```diff
@@ -142,6 +143,12 @@ function objectRestore(metadata, mdUtils, userInfo, request, log, callback) {
             if (isObjectRestored) {
                 monitoring.promMetrics(
                     'POST', bucketName, '200', 'restoreObject');
+                // If quotas are evaluated, increasing the quota duration should
+                // not affect the utilization metrics.
+                if (request.clearQuotaUpdatesForAPI?.[mdValueParams]) {
```

Comment:
* the name of this attribute should be improved, not very clear at the moment
* I don't see where this variable is set... 🧐
* since this is set in each request, I guess we should be able to directly set the appropriate callback (which would be cleaner IMO)... but then I did not see how it is initalized or added to the request

Reply:
> since this is set in each request, I guess we should be able to directly set the appropriate callback

I am not sure we can, the variable is set during the quota evaluation, and each API has custom ways to call their final callbacks, depending on the HTTP verb, etc.